src/CopyrighFixer/CopyrighFixer/signer.cpp

```diff
@@ -7,13 +7,55 @@
 
 #include "signer.h"
 #include "CopyrighFixer/ifilemanager.h"
+#include <QDir>
 
 namespace CopyrighFixer {
 Signer::Signer() {
 
 }
 
-bool Signer::checkSign(const Config &objConf) {
+bool Signer::checkSign(const Config &objConf) {    
+
+    Config currConfig = objConf;
+
+    QDir currentFolder(currConfig.getSrcDir());
+    currentFolder.setFilter(QDir::Dirs | QDir::Files);
+
+    QFileInfoList folderItems(currentFolder.entryInfoList());
+
+    for (auto i_file: folderItems) {
+
+        if (i_file.fileName() == "." || i_file.fileName() == "..") {
+            continue;
+        }
+
+        if (!i_file.isDir()) {
+
+            IFileManager *currFM = searchFileByExt(i_file.suffix());
+            Signature objSing = currConfig.getSignVal();
+
+            if (currFM == nullptr) {
+                return false;
+            }
+
+            bool valRead = currFM->read(i_file.filePath(), objSing);
```

Comment:
please read a documentation of the read method.
This method should be fill objSing, but you rewrite configuration sign because before invoke this method you invoke **currConfig.getSignVal()**.